manager/assignment.md

```diff
@@ -0,0 +1,26 @@
+## Overview 
+
+This technical writing and editing project is part of your interview. It is designed to take between 1.5 - 2 hours.
```

Comment:
I wonder if we should add a sentence to explain the purpose of this exercise？ Something like...

```suggestion
As a manager of product education engineering, you are accountable for ensuring the content quality which involves reviewing and providing clear feedback on your team's work. Therefore, this technical writing and editing project is part of your interview. It is designed to take between 1.5 - 2 hours.
```

Something to _hint_ that we're looking for the candidate's ability to coach the team. 

But if you think the recruiter will communicate that to candidates, please ignore this suggestion.

The instruction looks good!